MEMENTO/Basic_Camera/code.py

```diff
@@ -1,7 +1,23 @@
 # SPDX-FileCopyrightText: Copyright (c) 2023 john park for Adafruit Industries
 #
 # SPDX-License-Identifier: MIT
-''' simple point-and-shoot camera example. No bells! Zero whistles! '''
+""" 
```

Comment:
There is a trailing space on line 4, the first `"""` line.
```suggestion
"""
```

Reply:
thanks!